otelcol/command_components.go

```diff
@@ -24,13 +24,19 @@ type componentWithStability struct {
 	Stability map[string]string
 }
 
+type componentWithOutStability struct {
```

Comment:
"without" is one word.
```suggestion
type componentWithoutStability struct {
```